lib/streamlit/runtime/scriptrunner/script_runner.py

```diff
@@ -23,26 +23,26 @@
 from blinker import Signal
 
 from streamlit import config
-from streamlit import magic
 from streamlit import source_util
 from streamlit import util
 from streamlit.error_util import handle_uncaught_app_exception
-from streamlit.in_memory_file_manager import in_memory_file_manager
 from streamlit.logger import get_logger
 from streamlit.proto.ClientState_pb2 import ClientState
 from streamlit.proto.ForwardMsg_pb2 import ForwardMsg
-from streamlit.state import (
+from streamlit.runtime.in_memory_file_manager import in_memory_file_manager
+from streamlit.runtime.uploaded_file_manager import UploadedFileManager
+from streamlit.runtime.state import (
     SessionState,
     SCRIPT_RUN_WITHOUT_ERRORS_KEY,
     SafeSessionState,
 )
-from streamlit.uploaded_file_manager import UploadedFileManager
-from .script_run_context import ScriptRunContext, add_script_run_ctx, get_script_run_ctx
+from . import magic
```

Comment:
`magic.py` now lives inside the `scriptrunner` packages (which is the only code that uses it)